dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -642,8 +646,7 @@ void DAGStorageInterpreter::buildLocalStreams(DAGPipeline & pipeline, size_t max
         if (region_num == 0)
             continue;
         /// calculate weighted max_streams for each partition, note at least 1 stream is needed for each partition
```

Comment:
Remove this comment since it is out of date.

Reply:
done